modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/inquiry_details.rb

```diff
@@ -87,11 +87,11 @@ def handle_others_inquiry(inquiry_params, inquiry_details)
 
         def handle_family_inquiry(inquiry_params, inquiry_details)
           if inquiry_params[:is_question_about_veteran_or_someone_else] == 'Veteran' &&
-             inquiry_params[:their_relationship_to_veteran]
+             inquiry_params[:more_about_your_relationship_to_veteran]
             build_inquiry_details(
               inquiry_details: inquiry_details,
               inquiry_about: 'On Behalf of a Veteran',
-              veteran_relationship: inquiry_params[:their_relationship_to_veteran]
+              veteran_relationship: inquiry_params[:more_about_your_relationship_to_veteran]
             )
           elsif inquiry_params[:is_question_about_veteran_or_someone_else] == 'Someone else' &&
                 inquiry_params[:their_relationship_to_veteran]
```

Comment:
Should `their_relationship_to_veteran` be used here?

Reply:
No with the flow of the application. I'll be getting `more_about_your_relationship_to_veteran` and not `their`